cmake/std/atdm/cee-rhel6/custom_builds.sh

```diff
@@ -16,10 +16,18 @@ elif [[ $ATDM_CONFIG_BUILD_NAME == *"clang-5.0.1-openmpi-4.0.2"* ]] \
   || [[ $ATDM_CONFIG_BUILD_NAME == *"clang-5.0.1_openmpi-4.0.2"* ]] \
   || [[ $ATDM_CONFIG_BUILD_NAME == *"clang-5.0.1"* ]] \
   || [[ $ATDM_CONFIG_BUILD_NAME == *"clang-5"* ]] \
+  ; then
+  export ATDM_CONFIG_COMPILER=CLANG-5.0.1_OPENMPI-4.0.2
+  # Must list the default clang build last for correct matching of of defaults
+
+elif [[ $ATDM_CONFIG_BUILD_NAME == *"clang-9.0.1-openmpi-4.0.2"* ]] \
+  || [[ $ATDM_CONFIG_BUILD_NAME == *"clang-9.0.1_openmpi-4.0.2"* ]] \
+  || [[ $ATDM_CONFIG_BUILD_NAME == *"clang-9.0.1"* ]] \
+  || [[ $ATDM_CONFIG_BUILD_NAME == *"clang-5"* ]] \
```

Comment:
clang-9

Reply:
> clang-9

Thanks for the catch!  I have that fixed in commit 80d8a8d.